18_k8s_pvc_pv_svc_replicaset_liveness_readiness_11-oct-2022/README.md

```diff
@@ -10,3 +10,47 @@ and
 from ```.k8s``` folder of the project 
```

Comment:
Remove `Demo on` 
Specify the Linux distribution that is used to run all commands

Add Prerequisite section